pkg/types/types.proto

```diff
@@ -36,6 +36,7 @@ message Interface {
 	// library, regarding each type of link. Here is a non exhaustive
 	// list: "veth", "macvtap", "vlan", "macvlan", "tap", ...
 	string type = 7;
+	uint32 RawFlags = 8;
```

Comment:
Sorry to be picky but we should follow the protobuf syntax defined [here](https://developers.google.com/protocol-buffers/docs/proto#simple). Therefore, `RawFlags` should be `raw_flags` here.
But I noticed we forgot to enforce this for other fields of this structure, which is confusing...

Reply:
When use raw_flags, it can generate  different gzipped FileDescriptorProto  with rawFlags or RawFlags. Maybe we use rawFlags in order to be consistent with other fields. Next, I will submit another PR to follow the protobuf?

Reply:
I don't understand why the  continuous-integration failed ?

Reply:
The travis tests on ppc64le looks not stable today

Reply:
```
INFO: Forcing gometalinter version 
Installing:
  deadcode
  dupl
  errcheck
  gochecknoglobals
  gochecknoinits
  goconst
  gocyclo
  goimports
  golint
  gosec
  gotype
  gotypex
  ineffassign
  interfacer
  lll
  maligned
  misspell
  nakedret
  safesql
  staticcheck
  structcheck
  unconvert
  unparam
  varcheck
INFO: gometalinter args: '--exclude=vendor/.*" --tests --exclude=".*\.pb\.go" --disable-all --enable=gofmt --concurrency=2 --enable=misspell --enable=vet --enable=ineffassign --enable=gocyclo --cyclo-over=15 --enable=golint --deadline=600s --enable=structcheck --enable=unused --enable=staticcheck --enable=maligned --enable=varcheck --enable=unconvert'
```

```
gometalinter: error: unknown linters: unused
The command ".ci/static-checks.sh" failed and exited with 1 during .
```
I just met the issue in my host.  Maybe gometalinter removed unused.  I will take a look on it.

Reply:
I think we can just wait for https://github.com/kata-containers/tests/issues/1323 to fix this issue.

Reply:
Can we merge it ?  :-)